tree/dataframe/src/RDFNodes.cxx

```diff
@@ -675,6 +689,16 @@ void RLoopManager::RegisterCallback(ULong64_t everyNEvents, std::function<void(u
       fCallbacks.emplace_back(everyNEvents, std::move(f), fNSlots);
 }
 
+std::vector<std::string> RLoopManager::GetFiltersNames()
+{
+   std::vector<std::string> filters;
+   for (auto &filter : fBookedFilters) {
+      auto name = (filter->HasName() ? filter->GetName() : "Unnamed Filter");
+      filters.push_back(name);
+   }
```

Comment:
this could be done with a `std::transform`+lambda, i'm not sure which form would be more readable though...

Reply:
Maybe it's more readable as it is now?
```cpp
std::vector<std::string> RLoopManager::GetFiltersNames()
{
   std::vector<std::string> filters;

   auto pushNames= [](const RFilterBase* &filter){ return (filter->HasName() ? filter->GetName() : "Unnamed Filter"); };

   std::tranform(fBookedFilters.begin(), fBookedFilters.end(), pushNames);
   
   return filters;

}
```

Reply:
agreed